lib/configfiles/rhel7.xml

```diff
@@ -2,6 +2,38 @@
 <froxlor>
 	<distribution name="RHEL" version="7"
 		defaulteditor="/usr/bin/nano">
+      <defaults>
+        <propery>
```

Comment:
guess you mean "property". Why not just call it "setting", because that's what it is.

Reply:
yes, typo.
i can rename it, no problem